src/features/unified-share-modal/README.md

```diff
@@ -101,7 +101,19 @@ class USMExample extends React.Component {
                 const collaborators = this.contacts.slice();
 
                 const collaboratorsList = {
-                    collaborators,
+                    collaborators: this.contacts.map(contact => {
```

Comment:
Should the Storybook story also be updated?

Reply:
i didnt realize there was a storybook version of this. i can update